client/src/components/TaskList/TaskList.js

```diff
@@ -45,6 +63,16 @@ export default function TaskList() {
     const dragEndHandler = (result) => {
         //If result is invalid, return
         if (!result.destination || result.destination.index === result.source.index || !result) {
+            console.log("Invalid", result, toDelete.current);
+            // if (result.source.index !== undefined && toDelete.current) {
```

Comment:
do we still need this?

Reply:
Part of the implementation for drag to delete, not done properly yet. I wanna just leave it first